config/vufind/KohaRest.ini

```diff
@@ -151,3 +151,8 @@ extraFields = item-issue:acceptTerms:pickUpLocation
 ; config.ini defaults when Voyager is used for authentication.
 ;pattern = "numeric"
 ;hint = "Your optional custom hint can go here."
+
+; Show holds transaction history, default is set to false.
+[TransactionHistory]
+enabled = false
```

Comment:
This is otherwise fine, but should be commented-out by default so that enabling the history is a conscious decision.

Reply:
Err.. oops. Misread it.